rewards/classes/RewardsManager.py

```diff
@@ -244,9 +240,8 @@ def calculate_tree_distributions(self) -> RewardsList:
         all_dist_rewards = []
         for dist in tree_distributions:
             block = get_block_by_timestamp(self.chain, int(dist["timestamp"]))
-            token = dist["token"]["address"]
-            strategy = dist["id"].split("-")[0]
-            sett = self.get_sett_from_strategy(strategy)
```

Comment:
double check me on this but i think we can remove the get_sett_from_strategy function since it's only used here

Reply:
It's also used in `RewardsManager.py` line 278

Reply:
Removed